Create ja.json

mitekano23 opened the pull request:
Japanese translation was not available, so I added it. Please confirm.

mitekano23 committed:
Create ja.json

Japanese translation was not available, so I added it. Please confirm.

MindMeltMax commented:
Thanks, I have however updated the default translation on the latest release of fish nets.
If you'd be willing, could you add these as well, then I'll release a patch with the translations.

mitekano23 committed:
Update ja.json

I have translated the additional content and would appreciate your confirmation.
By the way, am I correct in understanding that TextureVariant uses numbers from 0 to 2?
If this recognition is correct, the translation is complete.

MindMeltMax commented:
Merged, will release a.s.a.p.
(Edit: It's up)
